listenbrainz/spotify_updater/spotify_read_listens.py

```diff
@@ -146,7 +147,7 @@ def make_api_request(user, spotipy_call, **kwargs):
                 # Rate Limit Problems -- the client handles these, but it can still give up
                 # after a certain number of retries, so we look at the header and try the
                 # request again, if the error is raised
-                time_to_sleep = e.headers.get('Retry-After', delay)
+                time_to_sleep = int(e.headers.get('Retry-After', delay))
```

Comment:
this could raise ValueError if for some reason it wasn't an int. add try/catch with a reasonable default. maybe 1 second?

Reply:
Since we are defaulting to delay if the header is missing, let's use that in case of an invalid header as well.